composer.json

```diff
@@ -67,8 +68,7 @@
         "illuminate/support": "self.version",
         "illuminate/translation": "self.version",
         "illuminate/validation": "self.version",
-        "illuminate/view": "self.version",
-        "tightenco/collect": "self.version"
```

Comment:
This is there to prevent `tightenco/collect` to be added at all if any of the project dependencies requires it. Removing this line would allow `tinghtenco/collect` to be installed and potentially causing mismatch `Illuminate\Support\Collection` to be available.